fastlane/report.xml

```diff
@@ -5,42 +5,14 @@
 -&gt; KlaviyoSwift (1.5.0)&#10;    - NOTE  | xcodebuild:  note: Using new build system&#10;    - NOTE  | xcodebuild:  note: Using codesigning identity override: -&#10;    - NOTE  | xcodebuild:  note: Build preparation complete&#10;    - NOTE  | [iOS] xcodebuild:  note: Planning&#10;    - NOTE  | [iOS] xcodebuild:  note: Building targets in dependency order&#10;    - NOTE  | [iOS] xcodebuild:  warning: The macOS deployment target 'MACOSX_DEPLOYMENT_TARGET' is set to 12.5, but the range of supported deployment target versions is 10.9 to 12.3.99.&#10;    - WARN  | xcodebuild:  /Users/noah.durell/code/klaviyo-swift-sdk/Sources/KlaviyoSwift/Klaviyo.swift:501:29: warning: 'archiveRootObject(_:toFile:)' was deprecated in iOS 12.0: Use +archivedDataWithRootObject:requiringSecureCoding:error: and -writeToURL:options:error: instead&#10;    - WARN  | xcodebuild:  /Users/noah.durell/code/klaviyo-swift-sdk/Sources/KlaviyoSwift/Klaviyo.swift:511:29: warning: 'archiveRootObject(_:toFile:)' was deprecated in iOS 12.0: Use +archivedDataWithRootObject:requiringSecureCoding:error: and -writeToURL:options:error: instead&#10;    - WARN  | xcodebuild:  /Users/noah.durell/code/klaviyo-swift-sdk/Sources/KlaviyoSwift/Klaviyo.swift:542:45: warning: 'unarchiveObject(withFile:)' was deprecated in iOS 12.0: Use +unarchivedObjectOfClass:fromData:error: instead&#10;&#10;[!] KlaviyoSwift did not pass validation, due to 3 warnings (but you can use `--allow-warnings` to ignore them).&#10;You can use the `--no-clean` option to inspect any issue.&#10;" />
```

Comment:
Do we want to deal with this failure message in the report before we merge this in?

Reply:
I dealt with that warning in another PR. So it should go away once that's merged but fair maybe I don't need to commit this? 🤔

Reply:
Ya I think its fine to leave this out. Doesn't exactly seem critical to anything